.github/workflows/ci.yml

```diff
@@ -0,0 +1,163 @@
+
+      - name: Build Python package
+        run: poetry run maturin develop --release
+
+      - name: pytest
+        run: poetry run pytest tests
+
+      - name: Benchmark
+        run: poetry run pytest benchmarks --compare
+
+  publish-rust:
+    if: startsWith(github.ref, 'refs/tags/')
+    needs: build
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+
+      - uses: actions-rs/toolchain@v1
+        with:
+          toolchain: nightly
+          override: true
+
+      - name: Publish on crates.io
+        run: |
+          cargo login ${{ secrets.CARGO }}
+          cargo publish
+
+  publish-python:
+    needs: build
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: [3.5, 3.6, 3.7, 3.8]
+        os: [ubuntu-latest, macos-latest, windows-latest]
+        exclude:
+          # There is a known issue where Python C extensions
+          # can not be built on Python 3.5 on Windows.
+          # https://github.com/actions/virtual-environments/issues/34
+          # Therefore, ujson can't be compiled and the build would fail.
+          - os: windows-latest
+            python-version: 3.5
+    steps:
+      - uses: actions/checkout@v1
+
+      - uses: actions/setup-python@v1
+        with:
+          python-version: ${{ matrix.python-version }}
+
+      - name: Install latest nightly
+        uses: actions-rs/toolchain@v1
+        with:
+          toolchain: nightly
+          override: true
+
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install poetry
+          poetry install
+
+      - name: Build Python package
+        run: poetry run maturin build --release --no-sdist --strip --interpreter python${{matrix.python_version}}
+
+      - name: List wheels
+        if: matrix.os == 'windows-latest'
+        run: dir target\wheels\
+
+      - name: List wheels
+        if:  matrix.os != 'windows-latest'
+        run: find ./target/wheels/
+
+      # Note: Windows doesn't support glob
+      # https://stackoverflow.com/a/52481267/270334
+      - name: Install wheels
+        if:  matrix.os == 'windows-latest'
+        run: pip install --find-links=target\wheels hyperjson
+
+      - name: Install wheels
+        if:  matrix.os != 'windows-latest'
+        run: pip install target/wheels/hyperjson*.whl
+
+      - name: Release
+        uses: softprops/action-gh-release@v1
+        if: startsWith(github.ref, 'refs/tags/')
+        with:
+          files: target/wheels/hyperjson*.whl
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+      # - name: Upload wheel
+      #   uses: actions/upload-artifact@v1
+      #   with:
+      #     path: target/wheels/
+
+      - name: PyPi publish
+        if: startsWith(github.ref, 'refs/tags/')
+        env:
+          MATURIN_PASSWORD: ${{ secrets.PYPI }}
+        run: poetry run maturin publish -b cffi --username __token__
```

Comment:
Why the `-b cffi`?

Reply:
Uhm... no good reason. I think I copy-and-pasted that from another pipeline. Shall I just remove it and it will figure it out by itself or shall I use a specific parameter here?

Reply:
I'm pretty sure that `-b cffi` will cause broken wheels; In general you need to use the same options for publish as for build with the exception that publish sets `--release --strip` by default.